eth_abi/utils/numeric.py

```diff
@@ -12,31 +10,13 @@
 )
 
 
-if sys.version_info.major == 2:
-    import struct
+def int_to_big_endian(value):
```

Comment:
I think we can just use the version of this from `eth-utils` and not have it in this library at all